README.md

```diff
@@ -90,15 +92,35 @@ git clone https://github.com/emmanuellmhango/railsblogapp.git
 
 Install this project with:
 
-cd railsblogapp
+```sh
+  cd railsblogapp
+  npm install
+  bundle install
+  rails db:create
+  rails db:migrate
+```
 
 ### Usage
 
 To run the project, execute the following command:
 
-`
+```sh
   rails server
-`
+```
+
+### Test
+
+To test the project, execute the following command:
+
+```sh
+  rspec
+```
+
+### Deployment
+
+> Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 👥 Author <a name="authors"></a>
```

Comment:
- [ OPTIONAL]
Since it is a collaboratibe project, you can consider adding the  contact information of both team members.